stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channels/ChannelsFragment.kt

```diff
@@ -35,22 +46,58 @@ class ChannelsFragment : Fragment() {
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        viewModel.bindView(binding.channelsListView, viewLifecycleOwner)
-        viewModel.state.observe(viewLifecycleOwner) {
-            binding.channelsLoadingView
-                .loadingViewContainer
-                .isVisible = it is ChannelsViewModel.State.Loading
+        setupOnClickListeners()
+        binding.channelsView.apply {
+
+            val loadingView = layoutInflater.inflate(
+                R.layout.channels_loading_view,
+                view as ViewGroup,
+                false
+            )
+
+            setLoadingView(loadingView, FrameLayout.LayoutParams(MATCH_PARENT, MATCH_PARENT))
+
+//            testViewHolderAPI()
```

Comment:
Left this in for easy testing / verification that custom view holders can be used